### PR TITLE
[feature] Add .bazelrc parsing & basic completion| #BAZEL-1186

### PR DESCRIPTION
**Pull request by mtoader**
This adds basic parsing plumbing and some completion for commands and existing configs.

### CODE REVIEW

File: plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/elements/BazelrcElementTypes.kt
Comment (agluszak) [inline plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/elements/BazelrcElementTypes.kt:12]: The RHS is the same in both cases

**Review by agluszak — COMMENTED** [review]
Could you please merge the latest master? We had a problem with a failing test

**Comment by abrams27** [comment]
@mpiekutowski can you take a look?

File: plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/completion/BazelrcCompletionContributor.kt
Comment (abrams27) [inline plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/completion/BazelrcCompletionContributor.kt:34]: can you please create a ticket on our youtrack for that?

**Comment by agluszak** [comment]
> @mpiekutowski can you take a look?

He did already :)

**Review by mpiekutowski — APPROVED** [review]
Everything keeps the convention of support for starlark, so looks good to me

File: plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/elements/BazelrcElementTypes.kt
Comment (mtoader) [reply plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/elements/BazelrcElementTypes.kt:12]: Yea. I wanted to default to something known for now. I'm still working on a a full PsiTree structure (I want to probably make the flag parsing a little better so I would need more nodes). 

This is temporary.

File: plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/completion/BazelrcCompletionContributor.kt
Comment (mtoader) [reply plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/completion/BazelrcCompletionContributor.kt:34]: Filed [BAZEL-1195](https://youtrack.jetbrains.com/issue/BAZEL-1195/Need-a-nice-way-to-know-what-flags-are-correct-for-a-particular-bazel-version)

**Comment by mtoader** [comment]
@mpiekutowski @agluszak I merged main here. Will this be landed automatically or do i need to ping someone with write access ?

**Comment by agluszak** [comment]
> @mpiekutowski @agluszak I merged main here. Will this be landed automatically or do i need to ping someone with write access ?

I'll merge it once the tests pass

**Comment by agluszak** [comment]
Hmm, it looks like the tests are still broken. @hb-man any clue?

**Comment by mtoader** [comment]
> Hmm, it looks like the tests are still broken. @hb-man any clue?

These seem unrelated to this change. Please let me know if I need to do something on my end.

**Comment by hb-man** [comment]
re-run flaky tests and merged, thanks for contribution @mtoader!
the tests failing i think an actual flaky problem in codebase, we are investigating, future commits made in the next week still might require manual intervention from our side (reruning the failing tests), sorry for inconvenience

**Pull request merged**
